typescript/sdk/src/middleware/account/InterchainAccount.ts

```diff
@@ -253,3 +254,74 @@ export async function deployInterchainAccount(
   );
   return interchainAccountApp.deployAccount(chain, config);
 }
+
+export function encodeIcaCalls(calls: CallData[], salt: string) {
+  return (
+    salt +
+    utils.defaultAbiCoder
+      .encode(
+        ['tuple(bytes32 to,uint256 value,bytes data)[]'],
+        [
+          calls.map((c) => ({
+            to: addressToBytes32(c.to),
+            value: c.value || 0,
+            data: c.data,
+          })),
+        ],
+      )
+      .slice(2)
+  );
+}
```

Comment:
why not use the typechain helpers?

Reply:
Which ones?

Reply:
I don't think there is a function that has the same encoding?